smf/nexus/multi-sled/config-partial.toml

```diff
@@ -55,6 +55,8 @@ blueprints.period_secs_execute = 60
 sync_service_zone_nat.period_secs = 30
 switch_port_settings_manager.period_secs = 30
 region_replacement.period_secs = 30
+instance_watcher.period_secs = 30
+instance_watcher.max_retries = 5
```

Comment:
Does this field still exist? I think no?

Reply:
ah, whoops, thanks for catching that!